.gitignore

```diff
@@ -0,0 +1 @@
+client/node_modules
```

Comment:
Just a final thing: there is already a .gitignore in the `client/` directory preventing `node_modules` from being commited, so this file is not necessary here. Please remove this file.